dll/opengl/Mesa-GLU_for_ReactOS.txt

```diff
@@ -0,0 +1,85 @@
+Here is a brief explanation on how to synchronize Mesa/GLU in ReactOS.
+
+Current included Mesa/GLU version is 9.0.0+.
+
+
+0) Mesa/GLU:
+
+
+0-1)  Mesa/GLU repository:
+
+https://cgit.freedesktop.org/mesa/glu/
+
+
+0-2) Inclusion in ReactOS code:
+
+Mesa/GLU 9.0.0 code was added in
+reactos/dll/opengl/glu32/
+by
+https://svn.reactos.org/svn/reactos?view=revision&revision=57621 (jgardou)
+
+
+1) Checking ReactOS included code against Mesa/GLU upstream code:
+
+
+1-1) Checking Mesa/GLU upstream feeds themselves:
+
+Diff of
+git clone https://anongit.freedesktop.org/git/mesa/glu.git
+git reset --hard glu-9.0.0
+and
+ftp://ftp.freedesktop.org/pub/mesa/glu/glu-9.0.0.tar.bz2
```

Comment:
Why? What's the goal/result of this?

Reply:
Because I didn't know which one was used. I wanted to find the closest source to what was imported.
The result is just below: I was right to check, because these 2 sources have different codes.